attorch/regularizers.py

```diff
@@ -14,10 +13,8 @@ class Laplace(nn.Module):
     """
     def __init__(self):
         super().__init__()
-
         self.conv = nn.Conv2d(1, 1, 3, bias=False, padding=1)
         self.conv.weight.data.copy_(torch.from_numpy(laplace()))
-        self.conv.weight.requires_grad = False
```

Comment:
why are we removing this? Shouldn't this stay there to ensure that one wouldn't try to learn the Laplace weights?